prisma/schema.prisma

```diff
@@ -15,25 +15,28 @@ enum Role  {
 
 model User {
     id String @id
+    name String
     createdAt DateTime @default(now())
     updatedAt DateTime @default(now())
     role Role @default(USER)
     ratings Rating[] @relation("Ratings")
+    ratingCount Int @default(0)
```

Comment:
If I recall correctly, PostgreSQL has a `COUNT` function, so I don't believe this field is needed.

Reply:
Commenting to bring attention to this, don't know if you glanced over it @ps251

Reply:
Right sorry. So I opted against using count because it's mentioned here that it is slow in postgres:
https://stackoverflow.com/questions/7943233/fast-way-to-discover-the-row-count-of-a-table-in-postgresql
But I guess it doesn't matter much either way. If you prefer, I can use count

Reply:
Oh, I didn't know that. I watched a video on how MySQL's `COUNT` function isn't slow (like many think it is) so I assumed the same was for PostgreSQL, but I guess I assumed incorrectly. Based on the research I've done and the post you linked it seems that it is actually wrong. I think keeping the `ratingCount` column would be a good choice